gameplay/src/TextBox.h

```diff
@@ -112,6 +112,11 @@ class TextBox : public Label
 
     virtual void addListener(Control::Listener* listener, int eventFlags);
 
+    /**
+     * Update the text being edited.
+     */
+    void setText(char const *text) override;
```

Comment:
Before I make that change, which clearly makes the code more error prone, I'd like to understand why you suggest this. "override" on overrides is a fantastic error catcher. "virtual" on things you intend to override is redundant and, without override, makes for easy mistakes. And "override" has been around in gcc, clang, and MSVC for a long time!